azure-pipelines.yml

```diff
@@ -27,34 +27,6 @@ resources:
       name: xamarin/yaml-templates
```

Comment:
These changes are not necessary, right?. Could you revert the changes?.

Reply:
well, i didnt change any of these. originally my base is main branch. see my other PR on main. @samhouts asked me to base on 4.8.0 and i dont know why but more changes are marked. It is probably because main has different yaml file than 4.8.0 branch.

Reply:
sorry this one went wrong. it was supposed to be from issue branch like this one https://github.com/xamarin/Xamarin.Forms/pull/11372/files
Can we delete this PR and concentrate on 11372?